src/modules/EventList/index.tsx

```diff
@@ -141,55 +142,72 @@ export const EventListUI = (props: EventListUIProps): ReactElement => {
                                         const primaryQuote = getPrimaryQuote(event.primaryCompany);
                                         const eventDate = DateTime.fromISO(event.eventDate);
                                         const audioOffset = (event.audioRecordingOffsetMs ?? 0) / 1000;
+                                        let divider = null;
```

Comment:
I think it might be good to write a `groupBy(array, fn)` that can be also be tested, and then group the events by date in the container and pass that grouped data into this component. `groupBy` will be useful in general, it adds tests for the logic to make sure we don't break it and keeps the logic in the container/utils instead of in the UI.